recipe-server/client/control_new/tests/state/extensions/test_reducers.js

```diff
@@ -1,29 +1,103 @@
-import { fromJS } from 'immutable';
+import { List, Map, is } from 'immutable';
 
 import {
-  EXTENSION_RECEIVE,
+  SESSION_INFO_RECEIVE,
+  SESSION_INFO_HISTORY_VIEW,
 } from 'control_new/state/action-types';
-import extensionsReducer from 'control_new/state/app/extensions/reducers';
+import sessionReducer from 'control_new/state/app/session/reducers';
 import {
   INITIAL_STATE,
-  ExtensionFactory,
-} from 'control_new/tests/state/extensions';
+} from 'control_new/tests/state/session';
 
 
-describe('Extensions reducer', () => {
-  const extension = ExtensionFactory.build();
-
+describe('Session reducer', () => {
   it('should return initial state by default', () => {
-    expect(extensionsReducer(undefined, { type: 'INITIAL' })).toEqual(INITIAL_STATE);
+    expect(sessionReducer(undefined, { type: 'INITIAL' })).toEqual(INITIAL_STATE);
+  });
+
+  it('should handle SESSION_INFO_RECEIVE', () => {
+    expect(sessionReducer(undefined, {
+      type: SESSION_INFO_RECEIVE,
+      history: [1, 2, 3],
+    })).toEqual({ history: new List([1, 2, 3]) });
   });
 
-  it('should handle EXTENSION_RECEIVE', () => {
-    expect(extensionsReducer(undefined, {
-      type: EXTENSION_RECEIVE,
-      extension,
-    })).toEqual({
-      ...INITIAL_STATE,
-      items: INITIAL_STATE.items.set(extension.id, fromJS(extension)),
+  describe('SESSION_INFO_HISTORY_VIEW', () => {
```

Comment:
Should these tests be in their own file, instead of adding on to `extensions/test_reducers.js`?